pkg/gencmd/generator.go

```diff
@@ -0,0 +1,189 @@
+package gencmd
+
+import (
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/izumin5210/clicontrib/pkg/clog"
+	"github.com/pkg/errors"
+	"github.com/shurcooL/httpfs/vfsutil"
+	"github.com/spf13/afero"
+
+	"github.com/izumin5210/grapi/pkg/cli"
+)
+
+type Generator interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Generator should have comment or be unexported